openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -108,13 +108,30 @@ const Carousel = {
                 cover.id = ocaid;
             }
 
-            const $el = $(`
-                <div class="book carousel__item">
+
+            const default_cover_url = 'https://openlibrary.org/images/icons/avatar_book-lg.png';
+
+            let book_cover = undefined
+            if (cover.id) {
+                book_cover = `
                     <div class="book-cover">
                         <a href="${work.key}">
-                            <img class="bookcover" src="//covers.openlibrary.org/b/${cover.type}/${cover.id}-M.jpg">
+                            <img class="bookcover" src="//covers.openlibrary.org/b/${cover.type}/${cover.id}-M.jpg?default=${default_cover_url}">
                         </a>
-                    </div>
+                    </div>`
+            } else {
+                book_cover = `
+                    <a href="${work.key}">
+                        <div class="carousel__item__blankcover bookcover">
+                            <div class="carousel__item__blankcover--title">${work.title}</div>
+                            ${work.author_name ? `<div class="carousel__item__blankcover--authors">${work.author_name}</div>` : ''}
+                        </div>
+                    <a/>`
+            }
+
+            const $el = $(`
+                <div class="book carousel__item">
+                    ${book_cover}
```

Comment:
```suggestion
                <div class="book carousel__item">
                    <div class="book-cover">
                        ${book_cover}
                    </div>
```
The `.book-cover` div is present when the cover image is missing.